contracts/Staking.sol

```diff
@@ -448,6 +469,19 @@ contract Staking is Autopetrified, ERCStaking, ERCStakingHistory, IStakingLockin
         emit Staked(_accountAddress, _amount, newStake, _data);
     }
 
+    function _unstakeUnsafe(address _from, uint256 _amount, bytes _data) internal {
```

Comment:
Thinking about this one more time, not sure why it is "unsafe", the way it's being used doesn't seem unsafe